metrics/web/test/dashboard/presentation/widgets/build_result_bar_test.dart

```diff
@@ -222,6 +222,32 @@ void main() {
       },
     );
 
+    testWidgets(
+      "displays the neutral graph indicator if the build status is in progress and the popup is opened",
+      (tester) async {
+        final buildResult = BuildResultViewModel(
+          buildResultPopupViewModel: BuildResultPopupViewModel(
+            date: DateTime.now(),
+            duration: Duration.zero,
+            buildStatus: BuildStatus.unknown,
+          ),
+          date: DateTime.now(),
+          duration: Duration.zero,
+          buildStatus: BuildStatus.inProgress,
+        );
+
+        await tester.pumpWidget(_BuildResultBarTestbed(
+          buildResult: buildResult,
+        ));
+
+        await _hoverBar(tester);
+
+        final graphIndicatorFinder = find.byType(NeutralGraphIndicator);
```

Comment:
```suggestion
        final neutralIndicatorFinder = find.byType(NeutralGraphIndicator);
```
Would it make this test a bit more clear?